source/macos/AdaptiveCards/AdaptiveCards/AdaptiveCard.swift

```diff
@@ -167,8 +167,11 @@ public struct InputFieldConfig {
     public let activeBorderColor: NSColor
     public let placeholderTextColor: NSColor
     public let multilineFieldInsets: NSEdgeInsets
+    public let errorMessageFont: NSFont
+    public let errorMessageTextAndBorderColor: NSColor
+    public let errorBackgroundColor: NSColor
```

Comment:
Would prefer to have separate `errorMessageTextColor` and `errorMessageBorderColor`.

Reply:
Infact it would be little more neater if we have a internal struct inside `InputFieldConfig` called `ErrorMessageConfig` with `font`, `textColor`, `borderColor` and `backgroundColor` and just one field `errorMessageConfig` in inputfieldConfig.